packages/directlinespeech/src/index.js

```diff
@@ -1,4 +1,4 @@
-/* global process */
+/* node/prefer-global/process */
```

Comment:
Is it correct ESLint syntax?

Reply:
yeah it's weird, but taking out the exception works too, so I'm doing that.